### ISSUE TITLE
Add API for Edits

### ISSUE THREAD

**Issue by sausheong** [sausheong/openai#2]
Given a prompt and an instruction, the model will return an edited version of the prompt. Add the APIs for for Edits.

**Comment by madhukaraman** [comment]
@sausheong can I do this ?

**Comment by sausheong** [comment]
Yes sure. You can send in pull requests for any one of these issues.
